applications/FluidDynamicsApplication/python_scripts/navier_stokes_compressible_explicit_solver.py

```diff
@@ -62,7 +62,8 @@ def GetDefaultParameters(cls):
                 "automatic_time_step" : true,
                 "CFL_number"          : 1.0,
                 "minimum_delta_time"  : 1.0e-8,
-                "maximum_delta_time"  : 1.0e-2
+                "maximum_delta_time"  : 1.0e-2,
+                "consider_compressibility" : true
```

Comment:
This will be always `true` so I wouldn't expose it to the user.

Reply:
Fixed. It is now enforced in `_CreateEstimateDtUtility`.